### ISSUE TITLE
poetry search with encode problems in python2.7

### ISSUE THREAD

**Issue by fabiocerqueira** [sdispater/poetry#52]
The search command only failed with python version 2.7.12.
I also test with Python 3.6.4 and it works fine on this version.

Command:
```
poetry search pagseguro
```

Output:
```
pagseguro (0.3.2)
 Pagseguro API v2 wrapper

pagseguro-api-v2 (0.0.2)
 Pagseguro API

django-oscar-pagseguro (0.0.4)
 Pagseguro integration for django-oscar

pagseguro-python (0.37)
 PagSeguro API v.2 client library

pagseguro-sdk (0.1.2)


  [UnicodeEncodeError]
  'ascii' codec can't encode characters in position 16-17: ordinal not in range(128)


search [-N|--only-name] [--] <tokens> (<tokens>)...```

**Comment by sdispater** [comment]
It should be fixed in the latest release (`0.8.4`).

Update `poetry` with `poetry self:update`.

**Comment by fabiocerqueira** [comment]
Thanks. I tested and It is fixed :D
